packages/framework/src/Resources/styles/admin/components/table/touch-always.less

```diff
@@ -3,6 +3,22 @@
     overflow: auto;
 
     &__in {
-        width: 900px;
+        width: 232px;
+
+        @media @query-sm {
```

Comment:
I don't see the reason for keeping this media query. Can you please remove it?

Reply:
Good point, removed.

Reply:
Well done :)